### ISSUE TITLE
Possible to update an existing node?

### ISSUE THREAD

**Issue by kagaw** [bumbeishvili/org-chart#329]
Is it possible to update an existing node? Currently I can only find addNode and removeNode methods.

**Comment by bumbeishvili** [comment]
Yes, it's dynamically updated one every render or expand & collapse, you can use either `nodeContent` for conditional html generation or `nodeUpdate` method to access dom element and update it
